Introduction_To_XenServer_Fuel_Plugin.md

```diff
@@ -9,30 +9,18 @@ As becoming part of the Big Tent, Mirantis Fuel has already made itself one of t
 
 ![XenServer Fuel plugin wizzard](https://github.com/openstack/fuel-plugin-xenserver/blob/master/doc/source/_static/fmwizard00.png?raw=true)
 
-Moreover, Mirantis Fuel even provides a control plane to let you customize the UI. As long as you follow the schema like [openstack.yaml](https://github.com/openstack/fuel-web/blob/master/nailgun/nailgun/fixtures/openstack.yaml), a brand new OpenStack release can be defined and exercised by Fuel. As shown in the above picture, we create our own release of OpenStack - "Liberty+Citrix XenServer on Ubuntu 14.04" - and upload it to Nailgun service, which contains all the business logic of the system. Interestingly, Mirantis seems to be also in fond of container technology and hosts most of major components inside the docker containers. Here comes the command.
-
-    dockerctl copy xs_release.yaml nailgun:/tmp/xs_release.yaml
-    dockerctl shell nailgun manage.py loaddata /tmp/xs_release.yaml
-    fuel rel --sync-deployment-tasks --dir /etc/puppet/
+Moreover, Mirantis Fuel even provides a control plane to let you customize the UI. As long as you follow the schema like [openstack.yaml](https://github.com/openstack/fuel-web/blob/master/nailgun/nailgun/fixtures/openstack.yaml), user can define their own OpenStack release. In Mirantis 8.0, user can even define their own resource type with [components.yaml](https://wiki.openstack.org/wiki/Fuel/Plugins#Component_compatibility_registry). In above screen shot, a hypervisor type "XenServer" is defined. And if you choose it, the subsequent wizzard and setting tabs will represent based on your choice and the incompatible list described in components.yaml. This feature is really useful because OpenStack setup is compilicated and there are many restrictions. With incompatible list, users will be prevented from making wrong choices.
 
-The reason to have an own release is we need to filter out incompatible user options. For example, as XenServer is chosen to be the hypervisor of the cluster, vCenter should be disabled. Another example is since VXLAN support hasn't been implemented for XenAPI, so we can only let user select VLAN for network segmentation.
-
-Except a self-defined OpenStack release, Mirantis also provide another approach for Fuel plugin to customize user interface. Within environment_config.yaml we define a bunch of attributes which finally will be rendered as the form shown below.
+In addition to provide restriction, Mirantis also provide a way to add input to web UI. In environment_config.yaml we define text fields to ask for XenServer credential information because we need to ssh into XenServer hosts to apply patches later on which finally will be rendered as shown below.
```

Comment:
Instead of "will be finally rendered as shown below" perhaps "which will be presented to the user in the web UI"